c++/samples/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+cmake_minimum_required (VERSION 2.8.12)
+
+project ("Cap'n'proto sample" CXX)
+
+# Needed to include generated Cap'n'proto headers
+set (CMAKE_INCLUDE_CURRENT_DIR ON)
+
+# Include CMake scripts
+include (FindPkgConfig)
+set (CMAKE_MODULE_PATH
+    "${PROJECT_SOURCE_DIR}/../cmake"
+    "${PROJECT_SOURCE_DIR}/cmake"
+)
+
+# Enable tests
+include(CTest)
+
+# Check for C++11 support
+include (C++11)
+CheckCXX11(FATAL_ERROR)
+
+# Add "-Wall" to compiler flags
+set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
+
+# Find Cap'n'proto
+find_package (Capnp REQUIRED)
+add_definitions(${CAPNP_DEFINITIONS})
+
+# Compile .capnp files
+capnp_generate_cpp(CAPNPSRC_addressbook CAPNPHDR_addressbook "${PROJECT_SOURCE_DIR}/addressbook.capnp")
+capnp_generate_cpp(CAPNPSRC_calculator CAPNPHDR_calculator "${PROJECT_SOURCE_DIR}/calculator.capnp")
+
+# Includes
+include_directories (
+    "${PROJECT_SOURCE_DIR}"
```

Comment:
Sorry, I meant that you could remove `"${PROJECT_SOURCE_DIR}"` and retain `CMAKE_INCLUDE_CURRENT_DIR` (or, replace it with `"${PROJECT_BINARY_DIR}"` and remove `CMAKE_INCLUDE_CURRENT_DIR`).